frame/bags-list/src/mock.rs

```diff
@@ -116,6 +123,11 @@ impl ExtBuilder {
 		sp_tracing::try_init_simple();
 		let storage = frame_system::GenesisConfig::default().build_storage::<Runtime>().unwrap();
 
+		// special cases used for `migrate` and `put_in_front_of` tests.
```

Comment:
preferably make them a `build_with_aux_data` or something like that, if possible.

Reply:
21aef98315ad14a36d380fa44dc9ec51b00887b4